gtk/src/light/gtk-3.0/_drawing.scss

```diff
@@ -60,14 +60,20 @@
     $_bc: if($c == $headerbar_bg_color, lighten($hb_pathbar_bg, 5%), $c);
     color: if($tc != $fg_color, transparentize($tc, .1), $fg_color);
     border-color: if($c != $base_color, transparent, $borders_color);
-    @if $variant == 'light' {
-      border-top-color: darken($_bc, if($c != $base_color, 12%, 20%));
+    @if $c!=$headerbar_bg_color {
+      @if $variant == 'light' {
+        border-top-color: darken($_bc, if($c != $base_color, 12%, 20%));
+      }
+      else {
+        border-top-color: darken($_bc, if($c != $base_color, 12%, 20%));
+      }
```

Comment:
it seems that we do the same in both light and dark branch. Are you sure we need this if/else?

Reply:
Seems pretty useless. I change it

Reply:
Ah no, I remember. For some reason, this would end up in this border top color for dark variant entries:

![image](https://user-images.githubusercontent.com/15329494/45624738-791d8b80-ba8b-11e8-98b9-132e6a096af4.png)

Whereas the current looks like this:

![image](https://user-images.githubusercontent.com/15329494/45624786-9b170e00-ba8b-11e8-909f-4d3a49ebe6ba.png)